src/components/primitive/text-area/text-area.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text, IconSvg } from 'components/primitive';
+import {
+  TextAreaContainer,
+  TextAreaLabel,
+  TextAreaWrapper,
+  StyledTextArea,
+  ErrorMessageContainer,
+  Icon,
+} from './text-area.component.styles';
+import { TextAreaComponentProps } from './text-area.component.props';
+
+export const TextArea = ({ value, label, isDisabled, error, ...rest }: TextAreaComponentProps) => {
```

Comment:
I don't think this component is exported in the `primitive/index.ts`

Reply:
Done, added the missing exports in the index.ts file.